openapi/p4pa-classification.openapi.yaml

```
@@ -204,8 +204,126 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ClassificationErrorDTO'
+  /classifications/{organizationId}/export:
+    get:
+      tags:
+        - classification
+      operationId: export
```

let's add a description to inform about the limit of 100k data
```suggestion
      operationId: export
      description "|
      The API returns a list of classifications.
      **Error:** If the size of the requested export is too large, the API will return an error."
```